src/lib/components/ratio/index.svelte

```diff
@@ -159,6 +234,11 @@
     grid-area: status;
     grid-template-columns: 1fr 1fr;
     text-align: center;
+
+    @include config.between('sm-page-break', 'lg-page-break') {
+      grid-template-rows: min-content;
```

Comment:
I would use `align-content: start` here, if the goal is not to stretch rows so they fill the grid.